lib/resources/example_json/r4_examples_encounter.rb

```diff
@@ -28,7 +28,31 @@ module Resources
               'url': 'custom-attribute-name'
             },
```

Comment:
It looks like the resource we used in this example was updated since we last used this. Since we are updating this now should we also updated the meta fields?

Reply:
[Updated to match the update :P](https://github.com/cerner/fhir.cerner.com/pull/438/commits/2400dbfe6f4f61bce9987e74cce923dd4a195410)

Reply:
+1